src/transform.ts

```diff
@@ -207,6 +208,35 @@ export interface WindowTransform {
   sort?: SortField[];
 }
 
+export interface JoinAggregateDef {
```

Comment:
FWIW, the similar thing for Aggregate is called `AggregateFieldDef`, so maybe `JoinAggregateFieldDef` for consistency?

Reply:
Done